app/views/fragments/checkout/basketPreview.scala.html

```diff
@@ -0,0 +1,15 @@
+<div class="basket-preview">
+    <h3 class="basket-preview__title">You've chosen:</h3>
+    <div class="basket-preview__image">
+        <img src="@controllers.CachedAssets.hashedPathFor("images/digital-pack.png")" alt="Digital Pack">
+    </div>
+    <div class="basket-preview__product">
+        Digital Pack
+        <span class="basket-preview__product__caption">
+            (Digital Edition / Guardian App Premium Tier)
```

Comment:
I'd assume so. This PR is only moving the markup, but I'll follow up on how we need to determine this,